formtranslate/api.py

```diff
@@ -104,5 +104,5 @@ def form_translate(input_data, operation, version='1.0'):
     stdout, stderr = p.communicate(input_data)
     exit_code = p.wait()
 
-    return ShellResult(stdout=stdout.decode(), stderr=stderr.decode(),
+    return ShellResult(stdout=stdout.decode('utf-8'), stderr=stderr.decode('utf-8'),
```

Comment:
@dannyroberts or @millerdev to confirm this seems right. i think it is, but will hold off. @kaapstorm assume you were able to test?